src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -247,8 +247,9 @@ interfaces:
       parameters:
       - shelf.name=Math
       - series_uuid.series_string=xyz3141592654
-      sample_args:
-      - shelf.name
+      attributes:
```

Comment:
FYI, the sample labeled "canonical" is the one where I'm trying to showcase all the samplegen features, as per @beccca 's request

Reply:
Ack. Since canonical is defined in the same method, I don't think we need to make any change to this PR right?